test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/UpdateAllProtectedFilesDuringReconnect.java

```diff
@@ -153,7 +153,7 @@ private HapiApiSpec updateAllProtectedFilesDuringReconnect() {
 								.path(ContractResources.VALID_BYTECODE_PATH),
 						contractCreate("testContract")
 								.bytecode("contractFile")
-								.autoRenewSecs(15)
+								.autoRenewSecs(1)
```

Comment:
This change is not correct! `.autoRenewSecs(15)` expects `.hasPrecheck(AUTORENEW_DURATION_NOT_IN_RANGE)` because there was an update to the `APP_PROPERTIES` when a node is disconnected
https://github.com/hashgraph/hedera-services/blob/8ce8e05f6d6a0c360b5ac0159f2f721d0b9c47c7/test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/UpdateAllProtectedFilesDuringReconnect.java#L95-L97

Reply:
Thanks @qnswirlds . Fixed by using as suggested 
```
fileUpdate(APP_PROPERTIES).payingWith(GENESIS)
								.overridingProps(Map.of("ledger.autoRenewPeriod.minDuration", "20"))
								.erasingProps(Set.of("minimumAutoRenewDuration")),
```
Passed test : https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1618360190343900